packages/core/parcel/package.json

```diff
@@ -10,7 +10,7 @@
     "type": "git",
     "url": "https://github.com/parcel-bundler/parcel.git"
   },
-  "bin": "src/bin.js",
+  "bin": "lib/bin.js",
```

Comment:
Commited by accident

Reply:
I never changed that, is some automated script changing that or something?

Reply:
Yes, `yarn build` would do this.

Reply:
Seems weird. That's why I like to copy my package.json to a dist/ folder which is .gitignored prior to doing any modifications for publishing.

Reply:
The problem is that the binary needs to be specified in package.json, if we just left it as `lib/bin.js`, the `parcel` binary wouldn't work in development.
The `lib` folders are ignored by git.

Reply:
The `parcel` binary needs to be specified in package.json, if we didn't do this, running `parcel` wouldn't work inside the monorepo.